connectors/src/connectors/slack/temporal/activities.ts

```diff
@@ -975,9 +975,18 @@ export async function deleteChannelsFromConnectorDb(
 
 export async function joinChannelAct(connectorId: ModelId, channelId: string) {
   const res = await joinChannel(connectorId, channelId);
+
   if (res.isErr()) {
     throw res.error;
   }
 
-  return res.value;
+  const { channel, result } = res.value;
+  if (result === "is_archived") {
+    logger.info(`Channel ${channelId} is archived, skipping sync`, {
```

Comment:
We only log message that are always the same to make it easy to search them in Datadog.
Also, first parameter should be the dictionary and the second one should be the message.

Reply:
Address in https://github.com/dust-tt/dust/pull/3505